README.md

```diff
@@ -1,122 +1,318 @@
-# Final Project Starter
+# AI CV Generator (Full Stack App)
 
-## Overview
+This is a full-stack resume/CV builder app that lets users create AI-enhanced resumes by filling out a multi-step form. The app integrates with the **Gemini API (Google AI)** to generate polished CV content tailored to job criteria.
```

Comment:
This is nice but I think we can expand it to include the `why` this project was built. This could even be as easy as copying some text from the presentation slide show. You have a great explanation there so we should use it :)